pkg/rke/k8s_rke_system_images.go

```diff
@@ -9721,6 +9763,48 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.28",
 		},
 		// Enabled out of band post v2.7.10
+		"v1.26.11-rancher2-3": {
```

Comment:
ACI was bumped to 6.0.4.1 for `v1.26.14-rancher1-1` in this PR. What is the reason for introducing `v1.26.11-rancher2-3` with ACI bumped to only 6.0.3.3?
Generally, we only add a single new version for a given minor as part of the KDM release with all changes rolled into it vs. introducing multiple versions.

Reply:
As discussed offline, an additional version is needed to provide an option to upgrade to ACI 6.0.3.3 vs. doing (what is a feature upgrade) to 6.0.4.1.
FYI @kinarashah @mitulshah-suse